packages/maskbook/package.json

```diff
@@ -12,14 +12,14 @@
     "@dimensiondev/kit": "=0.0.0-20210630045432-e05621c",
     "@dimensiondev/metamask-extension-provider": "3.0.6-20210519045409-1835d4d",
     "@dimensiondev/stego-js": "=0.11.1-20201027083223-8ab41be",
-    "@dimensiondev/uniswap-sdk": "=3.0.4-20210619092641-a270dc0",
     "@ethersproject/address": "^5.0.4",
     "@ethersproject/contracts": "^5.0.4",
     "@ethersproject/networks": "^5.0.4",
     "@ethersproject/providers": "^5.0.9",
     "@ethersproject/solidity": "^5.0.4",
     "@ethersproject/strings": "^5.1.0",
     "@masknet/contracts": "workspace:*",
+    "@masknet/constants": "workspace:*",
```

Comment:
plz do not import the package.

from `@masknet/web3-shared` import `get*Constants` or `use*Constants`